csrc/scheduler/registry.cpp

```diff
@@ -81,14 +81,19 @@ SchedulerRuntimeInfo::SchedulerRuntimeInfo(
       // find and push discontiguous stride
       int64_t dtype_size = dataTypeSize(input_tv->dtype());
       input_discontig_strides_[fusion_inp] = {};
-      int64_t dims = (int64_t)alloc_strides.size();
+      auto dims = static_cast<int64_t>(alloc_strides.size());
       int64_t expected_stride = 1;
       for (int64_t dim = dims - 1; dim >= 0; dim--) {
         auto size = alloc_sizes.at(dim);
+        auto stride = alloc_strides.at(dim);
+        // Skip broadcast dimensions because they don't affect contiguity.
         if (size <= 1) {
           continue;
         }
-        auto stride = alloc_strides.at(dim);
+        if (stride == 0) {
```

Comment:
I'm slightly uncomfortable with this one.

`stride == 0` is not something we cache on AFAIK. We should instead check for the TensorView's corresponding IterDomain being broadcast instead.

But we don't have to address it in this PR. Would be great if we can strip this one out and handle in a separate PR.